IntegrationTests/Runner/Environment.swift

```diff
@@ -0,0 +1,144 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+/// Encapsulates Python server configuration passed through ENV variable from  UITest runner to the app process.
+struct HTTPServerMockConfiguration: Codable {
+    /// Python server URL to record Logging requests.
+    var logsEndpoint: URL? = nil
+    /// Python server URL to record Tracing requests.
+    var tracesEndpoint: URL? = nil
+    /// Python server URL to record RUM requests.
+    var rumEndpoint: URL? = nil
+
+    /// Python server URLs to record custom requests, e.g. custom data requests
+    /// to assert trace headers propagation.
+    var instrumentedEndpoints: [URL] = []
+
+    /// Encodes this struct to base-64 encoded string so it can be passed in ENV variable.
+    var toEnvironmentValue: String {
+        let encoder = JSONEncoder()
+        let data = try! encoder.encode(self)
+        return data.base64EncodedString()
+    }
+
+    /// Decodes this struct from base-64 encoded string so it can be read from ENV variable.
+    fileprivate static func from(environmentValue: String) -> HTTPServerMockConfiguration {
+        let decoder = JSONDecoder()
+        let data = Data(base64Encoded: environmentValue)!
+        return try! decoder.decode(HTTPServerMockConfiguration.self, from: data)
+    }
+}
+
+internal struct Environment {
+    /// ENV variables shared between UITests and Example targets.
+    struct Variable {
+        static let testScenarioClassName = "DD_TEST_SCENARIO_CLASS_NAME"
+        static let serverMockConfiguration = "DD_TEST_SERVER_MOCK_CONFIGURATION"
+    }
+    /// Launch arguments shared between UITests and Example targets.
+    struct Argument {
+        static let isRunningUnitTests       = "IS_RUNNING_UNIT_TESTS"
```

Comment:
Some more cleanup can be done here around `isRunningUnitTests` - unlike this new Integration Tests project, the previous `Example` project was also launched in unit tests. With this constant we were preventing some app logic to run in that scenario. We no longer need it.

Reply:
I would like to keep it for now. The app could be run with Unit tests with the addition of IntegrationUnits, so this check could become handy :)